outputs.tf

```diff
@@ -8,7 +8,12 @@ output "route53resourcechange_role" {
   description = "IAM role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone."
 }
 
+output "sesmanagesuppressionlist_role" {
+  value       = aws_iam_role.sesmanagesuppressionlist_role
+  description = "IAM role that allows sufficient permissions to remove email addresses from the suppression list."
```

Comment:
Would it be more accurate to say it is a role that allows permissions to manage the suppression list? That aligns with the role name and the default description for the role.

Reply:
Yes.  See commit fb329c5.